app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationViewModel.kt

```diff
@@ -259,6 +268,25 @@ class CompleteRegistrationViewModel @Inject constructor(
         }
     }
 
+    private fun handleLoginResult(action: Internal.ReceiveLoginResult) {
+        handleDialogDismiss()
```

Comment:
Could we make a helper function that both `handleLoginResult` and `handleDialogDismiss` call.

Ideally, we would not have a `handle...` function called from anywhere but the `handleAction` or `handleInternalAction` methods.